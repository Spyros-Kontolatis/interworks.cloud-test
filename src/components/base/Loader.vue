<template>
  <div
    class="border-4 border-white border-b-primary rounded-full inline-block animate-spin absolute z-40"
  ></div>
</template>
