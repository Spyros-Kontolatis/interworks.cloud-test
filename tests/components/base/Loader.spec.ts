import { shallowMount } from "@vue/test-utils";
import Loader from "../../../src/components/base/Loader.vue";

describe("Loader.vue", () => {
  it("renders the component", () => {
    const wrapper = shallowMount(Loader);
    expect(wrapper.element).toMatchSnapshot();
  });
});
