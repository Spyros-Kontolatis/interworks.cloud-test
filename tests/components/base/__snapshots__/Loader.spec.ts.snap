// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader.vue renders the component 1`] = `
<div
  class="border-4 border-white border-b-primary rounded-full inline-block animate-spin absolute z-40"
/>
`;
